piton/ariane_setup.sh

```diff
@@ -49,7 +49,7 @@ echo
 
 echo "make sure that you source this script in a bash shell in the root folder of OpenPiton"
 
-if [ "$0" !=  "bash" ] && [ "$0" != "-bash" ]
+if [ "$0" !=  "bash" ] && [ "$0" != "-bash" ] && [ "$0" != "/bin/bash" ] && [ "$0" != "/usr/bin/bash" ]
```

Comment:
Thank god. People keep complaining about this. Is there a more elegant way to just check the last alphanumeric token is `bash` maybe?

Reply:
This stackoverflow suggests the variable `$BASH` will be set in bash? https://stackoverflow.com/questions/3327013/how-to-determine-the-current-interactive-shell-that-im-in-command-line